docs/feature_encoders.md

```diff
@@ -124,7 +124,7 @@ bool encoder_update_user(uint8_t index, bool clockwise) {
 }
 ```
 
-?> Media and mouse countrol keycodes such as `KC_VOLU` and `KC_WH_D` requires `EXTRAKEY_ENABLE = yes` and `MOUSEKEY_ENABLE = yes` respectively in user's `rules.mk` if they are not enabled as default on keyboard level configuration.
+?> Media and mouse control keycodes such as `KC_VOLU` and `KC_WH_D` requires `EXTRAKEY_ENABLE = yes` and `MOUSEKEY_ENABLE = yes` respectively in user's `rules.mk` if they are not enabled as default on keyboard level configuration.
```

Comment:
Please revert. 

This should be opened as a separate PR.

Reply:
@drashna done